recording/src/nextcloud/talk/recording/Participant.py

```diff
@@ -202,6 +209,50 @@ def __del__(self):
             # created in "/tmp".
             self.driver.quit()
 
+    def startChrome(self, width, height, env):
+        """
+        Starts a Chrome instance. Will use Chromium is Google Chrome is not installed.
+        :param width: the width of the browser window.
+        :param height: the height of the browser window.
+        :param env: the environment variables, including the display to start
+                    the browser in.
+        """
+
+        options = ChromeOptions()
+
+        # "webSocketUrl" is needed for BiDi; this should be set already by
+        # default, but just in case.
+        options.set_capability('webSocketUrl', True)
+
+        options.add_argument('--use-fake-ui-for-media-stream')
+
+        # Allow to play media without user interaction.
+        options.add_argument('--autoplay-policy=no-user-gesture-required')
+
+        options.add_argument('--kiosk')
+        options.add_argument(f'--window-size={width},{height}')
+        options.add_argument('--disable-infobars')
+        options.add_argument('--no-sandbox')
+        options.add_experimental_option("excludeSwitches", ["enable-automation"])
+
+        if disk_usage('/dev/shm').free < 2147483648:
+            self._logger.info('Less than 2 GiB available in "/dev/shm", usage disabled')
+            options.add_argument("--disable-dev-shm-usage")
+
+        if disk_usage('/tmp').free < 134217728:
+            self._logger.warning('Less than 128 MiB available in "/tmp", strange failures may occur')
+
+        service = ChromeService(
+            env=env,
+        )
+
+        self.driver = ChromeDriver(
+            options=options,
+            service=service,
+        )
+
+        self.bidiLogsHelper = BiDiLogsHelper(self.driver, self._parentLogger)
```

Comment:
Oh, I did not know that Chromium implemented already support for BiDi logs, good to know :-)

It seems that it was implemented for Chromium 110 (I did not find any information in the release notes, that guess is based on the results for [Chromium 109](https://wpt.fyi/results/webdriver/tests/bidi/log?sha=8ff38c54ff&label=master&max-count=1) and [Chromium 110](https://wpt.fyi/results/webdriver/tests/bidi/log?sha=6884bbba4e&label=master&max-count=1)), and I wonder if trying to run this on Chromium <= 109 would just cause no logs to be shown or a failure... Nevertheless, given that Chromium/Chrome version are usually kept up to date even on stable distributions I think it is safe to assume that Chromium >= 110 will be used (or that it will be feasible to update), so :+1: for using it unconditionally.